contracts/EncodeDAOCore.sol

```diff
@@ -110,16 +110,19 @@ contract EncodeDAOCore is ERC721URIStorage, AccessControl {
     {
         require(issueId <= _issueIds.current(), "IssueID is not valid");
         require(
-            pendingIssues[issueId].status == IssueStatus.Pending,
+            pendingIssues[issueId].status == IssueStatus.PENDING,
             "Issue is not pending"
         );
         require(
-            !votesOnIssues[issueId][msg.sender].voted,
+            votesOnIssues[issueId][msg.sender] == VoteStatus.NO_VOTE,
```

Comment:
I believe an issue is if the user has not voted yet, their field does not automatically contain `VoteStatus.NO_VOTE`
Maybe a workaround would be to remove `NO_VOTE` and just check that the field does not contain `ACCEPT` or `REJECT`?

Thoughts?

Reply:
Alternatively, may also check the field is equal to null, that could work too.